.github/workflows/build.yml

```diff
@@ -259,13 +259,14 @@ jobs:
           export BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)
           ./appveyor/docker_build/build_docker_image.sh
       - if: ${{ github.event_name == 'push' }}
-        name: Log in to Docker Hub
+        name: Login with Github Container registry
```

Comment:
Nit: `GitHub`.

Also, from what I see, it is called "GitHub Packages", not "GitHub Container".

Reply:
I humbly disagree: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry

Reply:
Fixed Github -> GitHub.